Content.Server/GameObjects/Components/Mobs/BuckleComponent.cs

```diff
@@ -59,6 +60,22 @@ private void BuckleStatus()
             }
         }
 
+        private bool IsContained()
```

Comment:
Unless the intent here is for this not to be recursive (for some reason) you can use `ContainerHelpers.IsInContainer(IEntity)` here.